clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -66,6 +72,7 @@ export async function activate(
   context: vscode.ExtensionContext,
 ): Promise<__ExtensionApi & __AnalysisApi> {
   DialectRegistry.clear();
+  Utils.setExtensionFolder(context.globalStorageUri.path);
```

Comment:
This will probably start with `/` on windows. It would be better to use the Uri directly as much as possible, or use the path version.